website/server/controllers/api-v3/tasks.js

```diff
@@ -204,6 +209,7 @@ api.getUserTasks = {
   async handler (req, res) {
     let types = Tasks.tasksTypes.map(type => `${type}s`);
     types.push('completedTodos');
+    types.push('allCompletedTodos');
```

Comment:
this can be condensed in a single call `types.push('completedTodos', 'allCompletedTodos')`